docs/source/changelog.rst

```diff
@@ -0,0 +1,347 @@
+  distributed projects)
+
+DataFrame
++++++++++
+
+- Refactor ``dask.dataframe`` to include a full empty pandas dataframe as
+  metadata.  Drop the ``.columns`` attribute on Series
+- Add Series categorical accessor, series.nunique, drop the ``.columns``
+  attribute for series.
+- ``read_csv`` fixes (multi-column parse_dates, integer column names, etc. )
+- Internal changes to improve graph serialization
+
+Other
+
+- Documentation updates
+- Add from_imperative and to_imperative functions for all collections
+- Aesthetic changes to profiler plots
+- Moved the dask project to a new dask organization
+
+0.7.6 / 2016-01-05
+------------------
+
+Array
++++++
+- Improve thread safety
+- Tree reductions
+- Add ``view``, ``compress``, ``hstack``, ``dstack``, ``vstack`` methods
+- ``map_blocks`` can now remove and add dimensions
+
+DataFrame
++++++++++
+- Improve thread safety
+- Extend sampling to include replacement options
+
+Imperative
+++++++++++
+- Removed optimization passes that fused results.
+
+Core
+++++
+- Removed ``dask.distributed``
+- Improved performance of blocked file reading
+- Serialization improvements
+- Test Python 3.5
+
+0.7.4 / 2015-10-23
+------------------
+
+This was mostly a bugfix release. Some notable changes:
+
+- Fix minor bugs associated with the release of numpy 1.10 and pandas 0.17
+- Fixed a bug with random number generation that would cause repeated blocks
+  due to the birthday paradox
+- Use locks in ``dask.dataframe.read_hdf`` by default to avoid concurrency
+  issues
+- Change ``dask.get`` to point to ``dask.async.get_sync`` by default
+- Allow visualization functions to accept general graphviz graph options like
+  rankdir='LR'
+- Add reshape and ravel to ``dask.array``
+- Support the creation of ``dask.arrays`` from ``dask.imperative`` objects
+
+Deprecation
++++++++++++
+
+This release also includes a deprecation warning for ``dask.distributed``, which
+will be removed in the next version.
+
+Future development in distributed computing for dask is happening here:
+http://distributed.readthedocs.org . General feedback on that project is most
+welcome from this community.
+
+0.7.3 / 2015-09-25
+------------------
+
+Diagnostics
++++++++++++
+- A utility for profiling memory and cpu usage has been added to the
+  ``dask.diagnostics`` module.
+
+Dataframe
++++++++++
+This release improves coverage of the pandas API. Among other things
+it includes ``nunique``, ``nlargest``, ``quantile``. Fixes encoding issues
+with reading non-ascii csv files. Performance improvements and  bug fixes
+with resample. More flexible read_hdf with globbing. And many more. Various
+bug fixes in ``dask.imperative`` and ``dask.bag``.
+
+0.7.0 / 2015-08-15
+------------------
+
+DataFrame
++++++++++
+This release includes significant bugfixes and alignment with the Pandas API.
+This has resulted both from use and from recent involvement by Pandas core
+ developers.
```

Comment:
The extra whitespace here is causing unintentional quoting in the rendered version